android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -324,6 +325,10 @@ public void onConfigurationChanged(Configuration newConfig) {
     super.onConfigurationChanged(newConfig);
     dismissKeyPreview(0);
     dismissSubKeysWindow();
+
+    RelativeLayout.LayoutParams params = KMManager.getKeyboardLayoutParams();
+    this.setLayoutParams(params);
```

Comment:
Will SystemKeyboard.java need this too?

Though I see it doesn't have the rest of `onConfigurationChanged()` either.

Reply:
This fixes both.  This is the `KMKeyboard` class, which is the base class of both the in-app and system keyboards.

Reply:
/facepalm